assets/newspack-ui/scss/_modals.scss

```diff
@@ -0,0 +1,77 @@
+.newspack-ui {
+	&__modal {
+		background: var( --newspack-ui-color-body-bg );
+		border-radius: var( --newspack-ui-border-radius );
+		max-width: var( --newspack-ui-modal-width-medium );
```

Comment:
Good call! This has been added in [5089ffe](https://github.com/Automattic/newspack-plugin/pull/2810/commits/5089ffe0df9b4e09d1bc381792e4ab9e2e234a4b).

Reply:
Thanks! Following the max-height change, shouldn't the overflow change from hidden to auto?

Reply:
@miguelpeixe I _think_ this should be handled by the [overflow added to the `.newspack-ui__modal__content` class](https://github.com/Automattic/newspack-plugin/pull/2810#discussion_r1430170027), right? That means the header bit is always visible, though, and only the content bit works -- that's definitely changeable if it'd be better to scroll everything! 

(Just let me know if I'm missing something here!)

Reply:
Currently the `.newspack-ui__modal__content` is not inheriting the max height:

<img width="454" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/820752/ac341965-fc74-48f5-be59-12119e965a66">

This should be fixed through flexboxes, adding `display: flex; flex-direction: column;` to `.newspack-ui__modal`.

Reply:
OHHHH thank you @miguelpeixe! I was testing the styles with the modal checkout, which was adding an inline height and tricking me into thinking the scrolling was working 😅 I see it now. 

I went the flexbox route in [210c7ba](https://github.com/Automattic/newspack-plugin/pull/2810/commits/210c7ba86fbd7d68a80c23a33fc31b8cb13c2060) and works well -- the only bit I'm waffling about is the modal footer. I'm wondering if it should also scroll instead of remaining in place (like it does with the current markup).

I'm thinking this may be something we can test/verify in action and see if it works or not. Based on the mockups, the only case a modal with a footer _may_ need to scroll would be on mobile, and maybe not even then. (I kind of faked the issue in the screenshot below, but the modal with a footer in the demo content isn't actually tall enough to cause this issue):

![image](https://github.com/Automattic/newspack-plugin/assets/177561/62fb869b-ba66-4aeb-ae07-f868755dad88)

Reply:
Hmmmm, I believe the footer should scroll along.

Reply:
Yeah, agreed -- it seems weird to treat it with the same importance as the header when it's more of a disclaimer space 🙂 

I've made an update in [2331596](https://github.com/Automattic/newspack-plugin/pull/2810/commits/2331596702837a5854dc47f593dd91f1f5881fdb) that makes the header sticky, and moves the overflow so everything else scrolls but the header stays put!